core/raftlease/fsm.go

```diff
@@ -583,9 +602,6 @@ func (c *Command) Validate() error {
 			return errors.NotValidf("setTime with model UUID")
 		}
 		if c.Lease != "" {
-			if c.Holder == "" {
```

Comment:
Why did you remove this?

Reply:
I'm pretty sure it's a rogue cut'n'pasto - it doesn't make any sense to have a holder on a setTime. I've checked with @manadart but haven't heard back yet.